environment.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+name: Thalassa
+
+channels:
+  - conda-forge
+
+dependencies:
+  - python=3.9
+  - proj<8
+  - datashader
+  - panel
+  - holoviews
+  - geoviews
+  - shapely=1.7.1
+  - ruamel.yaml
+  - black=21.11b0
```

Comment:
Cartopy > 0.20 causes bad performance. I guess that conda pulls cartopy 0.19 due to `proj<8` but I would rather have it as a primary dependency, too. We do `import cartopy` after all.

Reply:
Done